packages/cli/src/index.ts

```diff
@@ -1 +1,3 @@
 export { AutoRc } from "@auto-it/core";
+export * from "./parse-args";
```

Comment:
Is this ok? Should it have `as name` part, or it doesn't matter when importing it?

Reply:
This is fine!